frontend/src/components/CardsGrid/index.module.scss

```diff
@@ -0,0 +1,22 @@
+@import '../../components/ProjectCard/index.module.scss';
+
+$gap: 16px;
+
+.cardsGrid {
+  /* 
+    We can't simply use a css grid because it doesn't support centering elements if there are less
+    than 3 elements in the last row.
+  */
```

Comment:
```suggestion
```

I don't think this comment is necessary. Is there really a good reason to prefer `grid` over `flex` here? Grid always seemed more complicated to me, but I've haven't learned it yet. 

Though regarding the centering, I'm also not sure it looks good when there are only a few projects (e.g. on a user page), but we can leave that for now since the user page has not really been styled yet.